pyproject.toml

```diff
@@ -1,16 +1,28 @@
 [build-system]
 requires = ["setuptools", "wheel", "versioneer-518"]  # PEP 508 specifications.
 
-######
-# MyPy
-######
+##########
+## MyPy ##
+##########
 [tool.mypy]
+check_untyped_defs = true
+disallow_untyped_defs = true
+files = ['src']
 exclude = ["fidesops.migrations.*"]
-warn_unused_configs = true
-pretty = true
+ignore_missing_imports = true
```

Comment:
We don't want this one because it was causing real issues to be missed. It is instead now handled in the section below that ignores specific modules so we can be sure to only ignore ones that should be ignored.

```suggestion
```